spec/controllers/migration_controller_spec.rb

```diff
@@ -54,13 +55,25 @@
 
       it 'saves the story' do
         expect { post :create, params: params }.not_to raise_exception
-        expect(assigns(:story)).to be_valid
+        expect(assigns(:story).errors.full_messages).to be_blank
```

Comment:
There is a reason I switched to this: `be_valid` re-runs the validations, on a story that has been modified by the controller since it was validated there, whereas inspecting `errors` does not re-run the validations.